app/views/users/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% if @user.errors.any? %>
+<div id="error_explanation">
+  <h2>
+    <%= pluralize(@user.errors.count, "error") %> prohibited this article from being saved:
+  </h2>
+  <ul>
+    <% @user.errors.full_messages.each do |msg| %>
+    <li><%= msg %></li>
+    <% end %>
+  </ul>
+</div>
+<% end %>
+
+<!--
+<form method="post" action="/users" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for= "email"> Email </label>
+  <input id="email" name="user[email]" type="text">
+  <label for="username"> Username </label>  
+  <input id="username" name="user[username]" type="text">
+  <label for= "password"> Password </label>
+  <input id="password" name="user[password]" type="password">
+  <input name ="submit" type="submit" value= "Submit">
+</form>
+<%= form_tag('/users') do %>
```

Comment:
This form is throwing and error.

```
param is missing or the value is empty: user
user_params     params.require(:user).permit(:username,:password,:email)   end    end
-- | --



```

Reply:
Hello there,

Could you tell us how to replicate the bug?

Reply:
I commented the others forms went to users/new and tried to creat a new record .